website/www/site/content/en/documentation/programming-guide.md

```diff
@@ -4607,14 +4610,16 @@ window.
 
 Beam provides one data-driven trigger,
 <span class="language-java">`AfterPane.elementCountAtLeast()`</span>
-<span class="language-py">`AfterCount`</span>. This trigger works on an element
+<span class="language-py">`AfterCount`</span>
+<span class="language-go">`Trigger{Kind: ElementCountTrigger, ElementCount: 2}`</span>. This trigger works on an element
```

Comment:
Similar to the above we can probably simplify this down to `window.ElementCountTrigger` which matches how users will see it. We don't need to show the wrapping, that's what the code sample blocks are for.